recovery/elliptics_recovery/dc_server_send.py

```diff
@@ -0,0 +1,206 @@
+import logging
+import os
+import errno
+import time
+
+from elliptics_recovery.utils.misc import dump_key_data, load_key_data_from_file
+
+import elliptics
+
+log = logging.getLogger()
+
```

Comment:
This doesn't tell me how `on_server_send_fail()` behaves in bucket manager and server-send recovery classes and how they are related to each other.

Documentation in the code tells reader how particular **code** is supposed to be designed and how it is supposed to be working. Text is qdoc you reference above tells higher-level user how recovery works.